app/src/main/java/com/arygm/quickfix/model/messaging/ChatRepository.kt

```diff
@@ -3,6 +3,8 @@ package com.arygm.quickfix.model.messaging
 interface ChatRepository {
   fun init(onSuccess: () -> Unit)
 
+  fun getRandomUid(): String
```

Comment:
I don't think the name is very descriptive of what the function does:
```suggestion
  fun generateUid(): String
```

This is not a blocking comment however

Reply:
Missed that, sorry 😅